### PR TITLE
`helpdesk-faq`: service account and RBAC to get configmaps

### PR DESCRIPTION
**Pull request by smg247**
For: https://issues.redhat.com/browse/DPTP-3521

### CODE REVIEW

File: clusters/app.ci/assets/admin_helpdesk-faq.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+kind: List
```
Comment (bear-redhat) [inline clusters/app.ci/assets/admin_helpdesk-faq.yaml:1]: We probably don't need the List here

**Review by bear-redhat — CHANGES_REQUESTED** [review]

File: clusters/app.ci/assets/admin_helpdesk-faq.yaml (hunk 1 above)
Comment (deepsm007) [reply clusters/app.ci/assets/admin_helpdesk-faq.yaml:1]: Do we?

**Pull request merged**